crates/nu-command/src/commands/filesystem/rm.rs

```diff
@@ -25,7 +26,7 @@ impl WholeStreamCommand for Remove {
             )
             .switch("recursive", "delete subdirectories recursively", Some('r'))
             .switch("force", "suppress error when no file", Some('f'))
-            .rest(SyntaxShape::GlobPattern, "the file path(s) to remove")
+            .optional("files", SyntaxShape::GlobPattern, "the file path(s) to remove")
```

Comment:
You can pass more than one file to `rm`, so you'll need to keep this as `rest`

Reply:
I changed it to optional because `rm` currently needs at least one argument to run. I checked the Signature class and didn't find any optional + rest type, is there a way to do that?